File: mobile/src/main/java/com/siliconlabs/bledemo/log/DisconnectByButtonLog.java
package com.siliconlabs.bledemo.log;

import com.siliconlabs.bledemo.other.LogType;

public class DisconnectByButtonLog extends Log {

    public DisconnectByButtonLog(String deviceAddress) {
        setLogTime(getTime());
        setLogInfo(deviceAddress + " Disconnected on UI");
        setLogType(LogType.INFO); //malo wazne
        setDeviceAddress(deviceAddress);
    }

}

File: mobile/src/main/java/com/siliconlabs/bledemo/features/scan/browser/models/logs/DisconnectByButtonLog.kt
package com.siliconlabs.bledemo.features.scan.browser.models.logs


class DisconnectByButtonLog(deviceAddress: String) : Log() {
    init {
        logTime = getTime()
        logInfo = "$deviceAddress Disconnected on UI"
        super.deviceAddress = deviceAddress
    }
}
